coreneuron/gpu/nrn_acc_manager.cpp

```diff
@@ -89,10 +89,17 @@ void* cnrn_target_deviceptr(void* h_ptr) {
 #if defined(CORENEURON_ENABLE_GPU) && !defined(CORENEURON_PREFER_OPENMP_OFFLOAD) && defined(_OPENACC)
     return acc_deviceptr(h_ptr);
 #elif defined(CORENEURON_ENABLE_GPU) && defined(CORENEURON_PREFER_OPENMP_OFFLOAD) && defined(_OPENMP)
+    void *d_ptr = nullptr;
```

Comment:
Looks good but note that `cnrn_target_delete` also has `omp_get_mapped_ptr()`.
One question about `cnrn_target_delete()`: 

```
    omp_target_disassociate_ptr(h_ptr, device_id);
    auto* d_ptr = omp_get_mapped_ptr(h_ptr, device_id);
    omp_target_free(d_ptr, device_id);
```

Once we dissociate pointer, I don't think we can use `omp_get_mapped_ptr()` because it's removed from the present table.